.github/actions/update-velocitas-json/action.yml

```
@@ -0,0 +1,39 @@
+# Copyright (c) 2023-2024 Contributors to the Eclipse Foundation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: update-velocitas-json
+description: Update the project's velocitas JSON.
+inputs:
+  project_path:
+    description: Path to the directory containing the Velocitas project
+    required: true
+  template_path:
```

Both vars are named "path" but one is a dir (or folder) and the other one a file.
```suggestion
  project_dir_path:
    description: Path to the directory containing the Velocitas project
    required: true
  template_file_path:
```
... `_path` could be left off also.